check_emoji_sequences.py

```diff
@@ -76,9 +76,9 @@ def strip_vs_map(seq_map):
 
 def _check_no_vs(sorted_seq_to_filepath):
   """Our image data does not use emoji presentation variation selectors."""
-  for seq, fp in sorted_seq_to_filepath.iteritems():
+  for seq, fp in sorted_seq_to_filepath.items():
     if EMOJI_VS in seq:
-      print('check no VS: FE0F in path: %s' % fp)
+      print(f'check no VS: FE0F in path: {fp}')
```

Comment:
s/EMOJI_VS/EMOJI_VARIATION/

Don't use a const then hardcode it's value

Reply:
What do you mean with the hardcoded value, this one: `FE0F`?

Reply:
In the print statement.

Reply:
Removed it! 👍